### PR TITLE
leaf

### PR DESCRIPTION
**Pull request by ZakChrom**
🍁

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -366,6 +366,10 @@ async def on_message(message):
         icon = str(discord.utils.get(bot.get_guild(GUILD_ID).emojis, name="no_cat_throphy")) + " "
         await message.reply("**RoboTop**, I rate **you** 0 cats " + icon * 5)
 
+    if "leafbot" in message.author.name.lower() and "hmm... i would rate catbot" in message.content.lower():
```
Comment (milena-kos) [inline main.py:369]:
```suggestion
    if "leafbot" in message.author.name.lower() and "hmm... i would rate cat" in message.content.lower():
```

**Review by milena-kos — CHANGES_REQUESTED** [review]

**Comment by milena-kos** [comment]
uwunerd

**Comment by milena-kos** [comment]
owo i can do that

**Review by milena-kos — APPROVED** [review]
thumb up

**Comment by Floppy4** [comment]
image

**Comment by milena-kos** [comment]
that is, indeed, what this pr added
thanks for demonstrative imagery

**Pull request merged**